adls-plugins/src/main/java/co/cask/hydrator/plugin/sink/ADLSBatchSink.java

```diff
@@ -50,18 +50,21 @@
 import javax.annotation.Nullable;
 
 /**
- * ADLS batch sink
+ * A {@link BatchSink} that writes to Azure Data Lake Store(ADLS).
  */
 
 @Plugin(type = BatchSink.PLUGIN_TYPE)
-@Name("ADLSBatchSink")
+@Name("AzureDataLakeStore")
```

Comment:
This is a backwards incompatible change. Older pipelines that use the old sink will not be upgradeable. If a more descriptive display name is needed, you can set that in the widget json and the UI will display it with that name instead of the plugin name, which is really an identifier.